redirects/17.0.txt

```diff
@@ -112,6 +112,7 @@ applications/marketing/surveys/time_random.rst applications/marketing/surveys/qu
 applications/sales/crm/acquire_leads/generate_leads.rst applications/sales/crm/acquire_leads/email_manual.rst                                             #generate_leads --> email_manual
 applications/sales/crm/pipeline/multi_sales_team.rst applications/sales/crm/pipeline/manage_sales_teams.rst                                               # multi_sales_team --> manage_sales_teams
 applications/sales/point_of_sale/pricing/digital_labels.rst applications/sales/point_of_sale/pricing/electronic_labels.rst                                # digital_labels --> electronic_labels
+applications/sales/subscriptions/products.rst applications/sales/subscriptions.rst # subscriptions/products --> subscriptions
```

Comment:
No need to add a `17.0.txt` redirect line, the original redirect added to `16.0.txt` is already applied :)
```suggestion
```